package.json

```diff
@@ -42,6 +43,7 @@
     "eslint-config-airbnb-base": "14.0.0",
     "eslint-config-oclif": "3.1.0",
     "eslint-plugin-import": "2.18.2",
+    "fs-extra": "8.1.0",
```

Comment:
Did not know this lib 👍 

> I got tired of including mkdirp, rimraf, and ncp in most of my projects.

😄 (source: https://github.com/jprichardson/node-fs-extra#why)